src/java.desktop/macosx/native/libawt_lwawt/awt/ApplicationDelegate.m

```diff
@@ -475,6 +491,15 @@ + (void)_systemDidWake {
     [self _notifyJava:com_apple_eawt__AppEventHandler_NOTIFY_SYSTEM_WAKE];
 }
 
++ (void)_embeddedEvent:(NSNotification *)notification {
```

Comment:
@kevinrushforth @prrace 

Is '_embeddedEvent' too generic, does it need to be changed to something more inline with OpenURI e.g '_openURIEvent' ?

Reply:
As I noted in an earlier comment, I think the method name, _embeddedEvent , is fine, since it is private to this file.

Reply:
Also, while it is only used for openURI today, it could be used to pass additional messages in the future, so even if he were to change it, it shouldn't have "openURI" in the name.

Reply:
Good point. Agreed.